packages/core/src/anchor/anchor-processing-loop.ts

```diff
@@ -27,6 +30,11 @@ export class AnchorProcessingLoop {
    */
   readonly #anchorStoreQueue: NamedTaskQueue
   readonly #anchorPollingMetrics: TimeableMetric
+  /**
+   * Cache for the last time we polled a stream. Used to prevent polling the same stream more than once per pacing
+   * interval.
+   */
+  readonly #pollCache: Map<string, number> = new Map()
```

Comment:
Using a Map means the cache can grow unbounded.  Can we use an `LRUCache` instance instead?

Reply:
I was going to use an LRU cache after you suggested it earlier but had a concern about that that I forgot to ask.

Can the LRU cache get into a pathological state where a high ingress traffic rate can cause older entries to quickly age out, and now we're just polling for many CIDs constantly because they keep getting aged out?

e.g. With an LRU cache of 10K and an ingress TPS of 100, the oldest entries will get evicted every couple of minutes, and when the loop gets back around to them, it will just poll them again right away.

A large anchor store likely means an issue with the CAS, which means as soon as it comes back up, we might start slamming it.

Maybe a large enough cache would be sufficient? 100K?

Reply:
> Can the LRU cache get into a pathological state where a high ingress traffic rate can cause older
> entries to quickly age out, and now we're just polling for many CIDs constantly because they
> keep getting aged out?

Yes.

Reply:
> e.g. With an LRU cache of 10K and an ingress TPS of 100, the oldest entries will get evicted every couple of minutes, and when the loop gets back around to them, it will just poll them again right away.

If you have that many requests, then it will probably take more than a minute to get through them all, at which point you would want to poll them again anyway.

> Maybe a large enough cache would be sufficient? 100K?

I had been thinking more like 10k, but maybe 100k would be fine.  We just need to do some back-of-the-envelope math on how much memory this will take up.